### PR TITLE
Update Lesson_3_Let_Users_Mint_Your_NFT.md

### PR DESCRIPTION
**Pull request by ngnathan**
Fix typo "on" to "one" (end of sentence):

`Nice! Here you'll see my NFT here has "6 owners". You’ll also see it says “You own 1” which you’d see on your end as well as long as you minted one!`

### CODE REVIEW

File: JS_DAO/en/Section_2/Lesson_3_Let_Users_Mint_Your_NFT.md [hunk 1]
```diff
@@ -183,7 +183,7 @@ When you actually go to mint the NFT, Metamask will pop so you can pay gas. Once
 
 ![Untitled](https://i.imgur.com/PjjDSxd.png)
 
-Nice! Here you'll see my NFT here has "6 owners". You’ll also see it says “You own 1” which you’d see on your end as well as long as you minted on!
+Nice! Here you'll see my NFT here has "6 owners". You’ll also see it says “You own 1” which you’d see on your end as well as long as you minted one!
```
Comment (adilanchian) [inline JS_DAO/en/Section_2/Lesson_3_Let_Users_Mint_Your_NFT.md:186]: Can you tweak this to say:

Nice! Here you'll see my NFT has "6 owners". You’ll also see it says “You own 1” which you’d see on your end as well as long as you minted one!

**Review by adilanchian — CHANGES_REQUESTED** [review]
Just added one small change here :)

Thanks for the catch here!!

**Comment by ngnathan** [comment]
@adilanchian Thanks for reviewing!  Changes have been commited.

**Comment by adilanchian** [comment]
Hey @ngnathan -

Sorry for the late reply here! Looks great :). TY!

**Review by adilanchian — APPROVED** [review]

**Comment by adilanchian** [comment]
@all-contributors please add @ngnathan  for doc

**Pull request merged**